life_expectancy/tests/test_cleaning.py

```diff
@@ -1,18 +1,45 @@
 """Tests for the cleaning module"""
+from unittest import mock
+from pathlib import Path
 import pandas as pd
 
-from life_expectancy.cleaning import main
-from . import OUTPUT_DIR
+from life_expectancy.main import main
+from life_expectancy.data_loading import load_data, save_data
+from life_expectancy.data_cleaning import clean_data
 
+PROJECT_DIR = Path(__file__).parent.parent
+FILE_PATH = f"{PROJECT_DIR}/data/eu_life_expectancy_raw.tsv"
```

Comment:
Is the same suggestion Fernando did before, changing it to
`FILE_PATH = PROJECT_DIR / "data" / "eu_life_expectancy_raw.tsv"`